questionnaire.markdown

```diff
@@ -37,4 +37,5 @@ For your convenience, a copy of the questionnaire's questions is quoted here in
 >      Considerations" sections?
 > 17.  Do features in your specification enable origins to downgrade default
 >      security protections?
-> 18.  What should this questionnaire have asked?
+> 18.  How does your feature handle non-"[=Document/fully active=]" documents?
```

Comment:
Markdown doesn't support Bikeshed linking syntax, so maybe use a Markdown link here.

Reply:
I should make the Python script that generates this markdown file strip out Bikeshed syntax.

Reply:
Ah, right, this was automatically generated. I've removed the bikeshed syntax here and index.bs as well to not be blocked by the script change!